### ISSUE TITLE
Contributing suggestion

### ISSUE THREAD

**Issue by curquiza** [meilisearch/devrel#492]
Hello @meilisearch/devrel-team 👋
Here is a suggestion for contributing improvements to avoid this kind of issue we sometimes have: https://github.com/meilisearch/documentation/issues/1882

My suggestion is to
- update the default template for PRs, see my draft PR here https://github.com/meilisearch/.github/pull/11
- update our current [CONTRIBUTING.md](https://github.com/meilisearch/meilisearch/blob/main/CONTRIBUTING.md) to be more permissive and let people create PRs without creating necessarily issues for not-really-impactful changes like typo fixes. We can also add a warning a creating a PR without any approval for the change can be closed.

Currently, we sometimes have double notifications for not really important changes.
But most of all, I think it's important to remove the maximum of steps to let people contribute since we are not flooded by contributions yet. Closing PRs is of course something we allow.

I would love to have your opinion @dichotommy @brunoocasali on this question.

**Comment by brunoocasali** [comment]
I liked the idea @curquiza, and yeah, reducing the notifications burden is something I appreciated ❤️

**Comment by shivaylamba** [comment]
This is a really great idea! Especially with Hacktoberfest coming up!

**Comment by ferdi05** [comment]
Thanks for starting the discussion @curquiza .
I love the idea of removing any additional steps which are not super required.

I'm wondering how we could explain that contributors can sometimes skip the usual process: where to communicate about it and what to say? And also what are the criteria to determine if they can skip the process?

**Comment by curquiza** [comment]
> I'm wondering how we could explain that contributors can sometimes skip the usual process: where to communicate about it and what to say? And also what are the criteria to determine if they can skip the process?

Indeed, this is the part that is not easy to describe in the CONTRIBUTING...
I can try something, open a PR, and make you in review?

**Comment by ferdi05** [comment]
sure, happy to help if I can

**Comment by dichotommy** [comment]
Hello, I'm late to the party, but I agree with both of your suggestions @curquiza . Fewer notifications for us and fewer barriers for contributors, what's not to like! Happy to review once a PR has been created for the contributing guide.
